src/app/reducers/snapshots/redux/snapshot.reducers.ts

```diff
@@ -45,6 +47,17 @@ export function listReducer(
       };
     }
     case snapshot.LOAD_SNAPSHOT_RESPONSE: {
+      const sortByVolumeId = action.payload.reduce(
```

Comment:
groupByVolumeId or reduceByVolumeId

Should we duplicate snapshots in different states?
how do you handle the removal of the snapshot?